Octokit.Tests.Integration/Clients/IssuesClientTests.cs

```diff
@@ -230,6 +254,14 @@ public IssuesClientTests()
         Assert.True(retrieved.Any(i => i.Number == issue2.Number));
         Assert.True(retrieved.Any(i => i.Number == issue3.Number));
         Assert.True(retrieved.Any(i => i.Number == issue4.Number));
+        var issue = retrieved[0];
+        Assert.Equal(4, issue.Reactions.TotalCount);
+        Assert.Equal(0, issue.Reactions.Plus1);
+        Assert.Equal(0, issue.Reactions.Hooray);
+        Assert.Equal(0, issue.Reactions.Heart);
+        Assert.Equal(0, issue.Reactions.Laugh);
```

Comment:
As a memory jogger, `Issue` reactions were done first, then this PR is following up to do the other payload types (Issue comments, PR Review Comments, Commit Comments).

We actually had a problem with the merged PR for `Issue` reactions, where the `Plus1` and `Minus1` reaction types were broken, so we decided to change the integration tests to cover every reaction type using `foreach` loops.

But it looks like we only did integration tests for the single item `Get()` requests, and didnt actually think about the `GetAll()` multiple item calls, like this original test is covering.  

So in addition to fixing this test up so it works (ideally it should check each reaction type and the code could be alot cleaner using loops etc rather than explicitly creating 4 issues)... i think we also then need to add such a test for the `GetAll()` calls of the other payload types in this PR